packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3348,193 +3351,177 @@ export default class Meeting extends StatelessWebexPlugin {
-        } else if (isWrongReadyState) {
-          LoggerProxy.logger.warn(
-            `Meeting:index#closeRemoteTracks --> Error: MediaStreamTrack.readyState is ${track.readyState} for ${type}`
-          );
-        }
+    const stopStream = (stream: RemoteStream, type: string) => {
+      return Media.stopStreams(stream).then(() => {
+        triggerMediaStoppedEvent(type);
       });
     };
 
     return Promise.all([
-      stopTrack(remoteAudioTrack, EVENT_TYPES.REMOTE_AUDIO),
-      stopTrack(remoteVideoTrack, EVENT_TYPES.REMOTE_VIDEO),
-      stopTrack(remoteShare, EVENT_TYPES.REMOTE_SHARE),
+      stopStream(remoteAudioStream, EVENT_TYPES.REMOTE_AUDIO),
+      stopStream(remoteVideoStream, EVENT_TYPES.REMOTE_VIDEO),
+      stopStream(remoteShare, EVENT_TYPES.REMOTE_SHARE),
+      stopStream(shareAudioStream, EVENT_TYPES.REMOTE_SHARE_AUDIO),
     ]);
   }
 
   /**
-   * Stores the reference to a new microphone track, sets up the required event listeners
-   * on it, cleans up previous track, etc.
+   * Stores the reference to a new microphone stream, sets up the required event listeners
+   * on it, cleans up previous stream, etc.
    *
-   * @param {LocalMicrophoneTrack | null} localTrack local microphone track
+   * @param {LocalMicrophoneStream | null} localStream local microphone stream
    * @returns {Promise<void>}
    */
-  private async setLocalAudioTrack(localTrack?: LocalMicrophoneTrack) {
-    const oldTrack = this.mediaProperties.audioTrack;
+  private async setLocalAudioStream(localStream?: LocalMicrophoneStream) {
+    const oldStream = this.mediaProperties.audioStream;
 
-    oldTrack?.off(LocalTrackEvents.Muted, this.localAudioTrackMuteStateHandler);
-    oldTrack?.off(LocalTrackEvents.UnderlyingTrackChange, this.underlyingLocalTrackChangeHandler);
+    oldStream?.off(StreamEventNames.MuteStateChange, this.localAudioStreamMuteStateHandler);
+    oldStream?.off(LocalStreamEventNames.OutputTrackChange, this.localOutputTrackChangeHandler);
 
     // we don't update this.mediaProperties.mediaDirection.sendAudio, because we always keep it as true to avoid extra SDP exchanges
-    this.mediaProperties.setLocalAudioTrack(localTrack);
+    this.mediaProperties.setLocalAudioStream(localStream);
 
-    this.audio.handleLocalTrackChange(this);
+    this.audio.handleLocalStreamChange(this);
 
-    localTrack?.on(LocalTrackEvents.Muted, this.localAudioTrackMuteStateHandler);
-    localTrack?.on(LocalTrackEvents.UnderlyingTrackChange, this.underlyingLocalTrackChangeHandler);
+    localStream?.on(StreamEventNames.MuteStateChange, this.localAudioStreamMuteStateHandler);
+    localStream?.on(LocalStreamEventNames.OutputTrackChange, this.localOutputTrackChangeHandler);
 
-    if (!this.isMultistream || !localTrack) {
-      // for multistream WCME automatically un-publishes the old track when we publish a new one
-      await this.unpublishTrack(oldTrack);
+    if (!localStream) {
+      await this.unpublishStream(MediaType.AudioMain);
```

Comment:
as far as I can see, WCME does a replaceTrack() on a transceiver so it effectively unpublishes the old stream, so the comment from the old code (before this PR) still stands and we shouldn't be calling unpublishStream() on the send slot when we want to publish a different stream, just calling publishStream() with the new one should be enough

Reply:
Done: https://github.com/webex/webex-js-sdk/pull/3052